### DISCUSSION TITLE
How can convert language in a column to English? I can not do with Dbeaver. Any other tool that can help?

### DISCUSSION THREAD

**Discussion by jgrover21** [dbeaver/dbeaver#22435] [DBeaver Development]
How can convert language in a column to English? I can not do with Dbeaver. Any other tool that can help?

**Comment by LonwoLonwo** [comment]
Hello @jgrover21 

Please explain how exactly you want to convert language in a column. Need more details.

**Comment by jgrover21** [comment]
Hello, I will give you example.  I have data from Japan.  One column has comments in Japanese.  Ithink there are about 1 million rows.  I want to create a column and convert all comments to English.

**Reply by LonwoLonwo** [reply]
There is no translation function in DBeaver.

So, you need to use outside tools like Google Sheets, which can translate strings.

Steps to do it.

Open your table data in DBeaver and choose your column.
Click on the export in the context menu and choose export type. CSV, as an example.
![2024-01-12 15_56_18-Data Transfer](https://github.com/dbeaver/dbeaver/assets/45152336/95b82b9b-a6b0-4e18-9a64-b107b1dabef3)

Enable the "Selected columns" on the next settings page, continue, and finish your export.
![2024-01-12 15_56_32-Data Transfer](https://github.com/dbeaver/dbeaver/assets/45152336/e5a859bf-586f-41e7-af0b-6553ea71e9c2)

Import your data in Google Sheets. Translate data.

![Sheets-Code](https://github.com/dbeaver/dbeaver/assets/45152336/a82f59d5-359c-4644-a372-4d542d2b6a88)

Export your changed data to CSV format from Google Sheets.

Return to DBeaver, choose your table in the navigator view, and click Import.

![2024-01-12 15_56_47-DBeaver Ultimate 23 3 3 - film](https://github.com/dbeaver/dbeaver/assets/45152336/7ccce3ea-66b8-4658-850b-59f24045068b)

Choose the "create" status for the new column with translated values on the table mapping page.

**Comment by jgrover21** [comment]
Thank you! This is helpful. I will try steps
Maybe, Dbeaver should consider integration with Google sheets if possible.

Respectfully,

Jyoti Grover
